plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2231,15 +2217,13 @@ void api_base::send_transaction_gen(API &api, send_transaction_params_t params,
                   }
                   if (!retried) {
                      // we are still on main thread here. The lambda passed to `next()` below will be executed on the http thread pool
-                     auto yield        = abi_serializer::create_yield_function(fc::microseconds::maximum());
-                     auto abi_cache    = abi_serializer_cache_builder(api_base::make_resolver(api.db, std::move(yield))).add_serializers(trx_trace_ptr).get();
+                     auto abi_cache    = abi_serializer_cache_builder(make_resolver(api.db, api.abi_serializer_max_time, throw_on_yield::no)).add_serializers(trx_trace_ptr).get();
```

Comment:
Did you fix my issue too :-)?
https://github.com/AntelopeIO/leap/issues/1025

Reply:
Only in this one place, I believe there are other locations as well.

Reply:
well, thanks anyways!